releasenotes/README.md

```diff
@@ -10,6 +10,12 @@ If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx
 
 [Click here for the v48 release notes.](./v48.md)
 
+## 49.13.1 (October 8, 2020)
```

Comment:
Content looks good.  But please put the 49.13.1 notes after the 49.14.0 notes, just to keep them ordered.